LICENSE

```diff
@@ -206,13 +206,13 @@
 
 ====================================================================================================
 
-ODF TOOLKIT DEPENDENCIES: 
+TDF ODF TOOLKIT DEPENDENCIES: 

```

Comment:
hmm i don't see much value in pefixing TDF here ... at the ASF that was some sort of policy to prefix Apache to everything but it's not called "TDF LibreOffice" so why do it here?